linkerd/app/src/env.rs

```diff
@@ -249,6 +249,8 @@ const ENV_OUTBOUND_HTTP1_CONNECTION_POOL_IDLE_TIMEOUT: &str =
 
 const ENV_SHUTDOWN_GRACE_PERIOD: &str = "LINKERD2_PROXY_SHUTDOWN_GRACE_PERIOD";
 
+const ENV_PROXY_TRACING_SERVICE_NAME: &str = "LINKERD2_PROXY_TRACING_SERVICE_NAME";
```

Comment:
For the sake of consistency, we already have:
```
pub const ENV_TRACE_ATTRIBUTES_PATH: &str = "LINKERD2_PROXY_TRACE_ATTRIBUTES_PATH";
```

can we call this `LINKERD2_PROXY_TRACE_SERVICE_NAME` and group the definitions?

Reply:
Done